snap/snapcraft.yaml

```diff
@@ -205,6 +204,7 @@ parts:
     - libpixman-1-dev
     - libseccomp-dev
     - meson
+    - python3-venv
```

Comment:
This is new build dependency was first introduced in [8.1](https://wiki.qemu.org/ChangeLog/8.1#Build_Information).  You will see:
> - New Python build dependencies:
>   - venv (part of the standard library but sometimes placed in a separate package such as Debian's python3-venv).

Since we are moving from 8.0 to 8.2, we now need this to build qemu.

Reply:
Oh, great! So let's get this in!